src/UCTNode.cpp

```diff
@@ -207,10 +215,32 @@ void UCTNode::set_policy(float policy) {
     m_policy = policy;
 }
 
+float UCTNode::get_variance(float default_var) const {
+    return m_visits > 1 ? m_squared_diff / (m_visits - 1) : default_var;
+}
+
+float UCTNode::get_stddev(float default_stddev) const {
+    return m_visits > 1 ? std::sqrt(get_variance()) : default_stddev;
+}
+
 int UCTNode::get_visits() const {
     return m_visits;
 }
 
+float UCTNode::get_lcb(int color) const {
+    // Lower confidence bound of winrate.
```

Comment:
No need for the comment if named get_winrate_lcb()

Reply:
I left the comment there to explain what lcb means.